util.go

```diff
@@ -35,6 +36,16 @@ func ExitFail(format string, a ...interface{}) {
 	os.Exit(1)
 }
 
+func ShellOut(command *string, name string) {
+	if command != nil {
+		shell := exec.Command("/bin/sh", "-c", *command)
```

Comment:
Should it emit a message before running so we are aware what it just did?

I.e:
```
fmt.Printf("Running %s commands:\n %s", name, command)
```